omero/sysadmins/mapping-restricted-admins.txt

```diff
@@ -0,0 +1,48 @@
+Restricted administrators OMERO.web interface to server permissions mapping
+===========================================================================
+
+
+Summary
+-------
+
+OMERO allows you to create administrators with a subset of the full
+administrator privileges,
+see :doc:`/sysadmins/admins-with-restricted-privileges`.
+The OMERO.web user interface form for creation and editing of
+restricted administrators
+(see the :help:`create new users <sharing-data#admin>` section)
+collates the server-side privileges
+into fewer options and gives the options user friendly
+names. Here, the mapping of the OMERO.web options to the 
+server-side privileges is given. The server-side privileges
+are more granular and direct work with them is possible on the CLI,
+as described in :doc:`/sysadmins/cli/LightAdmins`.
```

Comment:
Yes, here https://github.com/openmicroscopy/ome-documentation/pull/1727/commits/456046a11d6cda280ddf199ae743f529d928fa7f - there is a bit of chicken and egg problem. Those tow PRs will not be able to live side by side.